### PR TITLE
Auto-generate README with tag information

### PR DESCRIPTION
**Pull request by malcyon**
It seems I've created a mess with having so many different image tags for the docker image (jdk-11, arm64, etc).

The idea here is to auto-generate the Readme using the format suggested in https://github.com/Graylog2/graylog-docker/pull/191, so the releaser doesn't have to manually update each tag in the README.  

Replace the static `VERSION` and `VERSION_FORWARDER_PACKAGE` files with a `version.yaml` file that the script can easily parse.

### CODE REVIEW

**Comment by malcyon** [comment]
@bernd @thll This is just an unfinished idea here. Any objections to this?

**Comment by malcyon** [comment]
Quick thoughts:

- Create a new `release.groovy` Jenkins pipeline for handling the Docker release. It would handle updating the `version.yml`, regenerating the README, committing the changes, and adding a new tag. All the stuff we do manually right now. 
- Have a second stage that checks out the new tag. 
- Move the logic from `jenkins.groovy` to build the Java 11 and ARM64 images into stage 2 of the release pipeline. 
- Add buildx commands to build `graylog`, `graylog-enterprise`, and `forwarder` into the release pipeline. This should be easy since the current logic for the ARM64 and Java11 images already mimics all the weird regex tag stuff we do.
- Now, all our official images are multiplatform, which reduces the need for all the extra `arm64` tags. Less confusion. 
- Now that we're building the images on our own CI servers, adding in Content Trust (https://github.com/Graylog2/graylog-docker/issues/23) in a later PR would be easier.

**Comment by malcyon** [comment]
Alright, I am going to put the release pipeline stuff in a separate PR and restrict this to just the README changes.

**Review by thll — APPROVED** [review]
LGTM

File: README.j2 [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+| Java Version  | Platform  | Tags  |
+|---|---|---|
+| OpenJDK 11  | `linux/amd64`, `linux/arm64`  | `{{ graylog.major_version }}.{{ graylog.minor_version }}-jre11`, `{{ graylog.major_version }}.{{ graylog.minor_version }}.{{ graylog.patch_version }}-jre11`, `{{ graylog.major_version }}.{{ graylog.minor_version }}.{{ graylog.patch_version }}-{{ graylog.release }}-jre11`  |
+| OpenJDK 8  | `linux/amd64` | `{{ graylog.major_version }}.{{ graylog.minor_version }}`, `{{ graylog.major_version }}.{{ graylog.minor_version }}.{{ graylog.patch_version }}`, `{{ graylog.major_version }}.{{ graylog.minor_version }}.{{ graylog.patch_version }}-{{ graylog.release }}` |
+| OpenJDK 8  | `linux/arm64` | `{{ graylog.major_version }}.{{ graylog.minor_version }}-arm64`, `{{ graylog.major_version }}.{{ graylog.minor_version }}.{{ graylog.patch_version }}-arm64`, `{{ graylog.major_version }}.{{ graylog.minor_version }}.{{ graylog.patch_version }}-{{ graylog.release }}-arm64` |
```
Comment (thll) [inline README.j2:39]: Do we have a `graylog.release` number at all? I though it was only for the Forwarder.

With the current version of `version.yml`
```
{{ graylog.major_version }}.{{ graylog.minor_version }}.{{ graylog.patch_version }}-{{ graylog.release }}-jre11
```
will render to 
```
4.2.2--jre11
```
which is obviously wrong.

**Review by thll — CHANGES_REQUESTED** [review]

File: README.j2 (hunk 1 above)
Comment (malcyon) [reply README.j2:39]: Oh my, you are right. Okay, it should be fixed now.

**Review by thll — DISMISSED** [review]
Looks good, but needs to be synchronized with the `4.2` branch now.

**Review by thll — APPROVED** [review]

**Pull request merged**